stable/sentry/README.md

```diff
@@ -66,6 +66,7 @@ Parameter                                            | Description
 `image.tag`                                          | Sentry image tag                                                                                           | `9.1.1`
 `imagePullPolicy`                                    | Image pull policy                                                                                          | `IfNotPresent`
 `imagePullSecrets`                                   | Specify image pull secrets                                                                                 | `[]`
+`sentrySecret`                                       | Specify fixed SENTRY_SECRET_KEY                                                                            | `nil`
```

Comment:
I would change this.
Something like:
`Specify SENTRY_SECRET_KEY. If isn't specified it will be generated automatically.`

Reply:
Done